dashboard/lib/services/lti.rb

```diff
@@ -16,12 +16,25 @@ def self.initialize_lti_user(id_token)
       user.name = custom_claims[:display_name] || custom_claims[:full_name] || custom_claims[:given_name]
       user.family_name = custom_claims[:family_name]
     end
+    email = custom_claims[:email]
+    data = {
+      email: email
+    }
     ao = AuthenticationOption.new(
       authentication_id: Policies::Lti.generate_auth_id(id_token),
       credential_type: AuthenticationOption::LTI_V1,
-      email: custom_claims[:email],
+      email: email,
+      data: data.to_json
     )
     user.authentication_options = [ao]
     user
   end
+
+  # Removes the cleartext email for LTI AuthenticationOptions
+  def self.remove_cleartext_email(auth_option)
+    data = auth_option.data_hash
+    data[:email] = ''
+    auth_option.data = data.to_json
+    auth_option.email = ''
```

Comment:
Why do we store the email both at the top level and in the `data` property?

Reply:
I was doing this so we could remember the original email the LMS gave us. That way we could only show the `email` form input if the LMS didn't provide an email. However, we are just going to always show the email now, so I'm going to drop this `data` work.